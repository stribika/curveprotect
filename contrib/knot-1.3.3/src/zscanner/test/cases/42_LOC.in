$ORIGIN	.
$TTL	1

; OK
@	LOC	1     N 1     E	0			; The simplest case
@	LOC	0 1   N 1     E 0			; Combination of parameters
@	LOC	0 0 1 N 1     E 0			; Combination of parameters
@	LOC	1     N 0 1   E 0			; Combination of parameters
@	LOC	1     N 0 0 1 E 0			; Combination of parameters
@	LOC	1     N 0 0 1 E 0m			; Combination of parameters
@	LOC	1     N 1     E 0 1			; Combination of parameters
@	LOC	1     N 1     E 0 1m			; Combination of parameters
@	LOC	1     N 1     E 0 0 1			; Combination of parameters
@	LOC	1     N 1     E 0 0 1m			; Combination of parameters
@	LOC	1     N 1     E 0 0 0 1			; Combination of parameters
@	LOC	1     N 1     E 0 0 0 1m		; Combination of parameters
@	LOC	0 0 0 N 0 0 0 E	-100000.00 0 0 0	; Minimal values
@	LOC	90 59 59.999 S 180 59 59.999 W 42849672.95m 90000000.00m 90000000.00m 90000000.00m	; Maximal values
@	LOC	0     S 0 0 0.001 W 0			; Float dd.ddd test
@	LOC	0     S 0 0 0.01  W 0			; Float dd.ddd test
@	LOC	0     S 0 0 0.1   W 0			; Float dd.ddd test
@	LOC	0     S 0 0 1.0   W 0			; Float dd.ddd test
@	LOC	0     S 0 0 10    W 0			; Float dd.ddd test
@	LOC	0     S 0     W 0 0.01			; Number to [mantisa,exponent] test
@	LOC	0     S 0     W 0 0.10			; Number to [mantisa,exponent] test
@	LOC	0     S 0     W 0 1.0			; Number to [mantisa,exponent] test
@	LOC	0     S 0     W 0 10			; Number to [mantisa,exponent] test
@	LOC	0     S 0     W 0 100			; Number to [mantisa,exponent] test
@	LOC	0     S 0     W 0 1000			; Number to [mantisa,exponent] test
@	LOC	0     S 0     W 0 10000			; Number to [mantisa,exponent] test
@	LOC	0     S 0     W 0 100000		; Number to [mantisa,exponent] test
@	LOC	0     S 0     W 0 1000000		; Number to [mantisa,exponent] test
@	LOC	0     S 0     W 0 10000000		; Number to [mantisa,exponent] test
@	LOC	\# 16 00 00 00 00 00000000 00000000 00000000	; Hexadecimal rdata
@	TYPE29	\# 16 00 00 00 00 00000000 00000000 00000000	; TYPE + Hexadecimal rdata
@	TYPE29	0     N 0     E 0			; TYPE
@	loc	0     N 0     E 0			; Type in lower-case

; KO
@	LOC
@	LOC						; Empty rdata
@	LOC	\# 0					; Hex empty rdata
@	LOC	\#					; Missing hex length
@	LOC	91 0 0 N 0 0 0 E 0 0 0 0		; Degree overflow
@	LOC	0 60 0 N 0 0 0 E 0 0 0 0		; Minute overflow
@	LOC	0 0 60 0 N 0 0 0 E 0 0 0 0		; Second overflow
@	LOC	0 0 0 N 181 0 0 E 0 0 0 0		; Degree overflow
@	LOC	0 0 0 N 0 60 0 E 0 0 0 0		; Minute overflow
@	LOC	0 0 0 N 0 0 60 E 0 0 0 0		; Second overflow
@	LOC	0 0 0 N 0 0 0 E 42849672.96 0 0 0	; Altitude overflow
@	LOC	0 0 0 N 0 0 0 E 42849673 0 0 0		; Altitude overflow
@	LOC	0 0 0 N 0 0 0 E -100000.01 0 0 0	; Altitude underflow
@	LOC	0 0 0 N 0 0 0 E -100001 0 0 0		; Altitude underflow
@	LOC	0 0 0 N 0 0 0 E 0 90000000.01 0 0	; Size overflow
@	LOC	0 0 0 N 0 0 0 E 0 90000001 0 0		; Size overflow
@	LOC	0 0 0 N 0 0 0 E 0 0 90000000.01 0	; HP overflow
@	LOC	0 0 0 N 0 0 0 E 0 0 90000001 0		; HP overflow
@	LOC	0 0 0 N 0 0 0 E 0 0 0 90000000.01	; VP overflow
@	LOC	0 0 0 N 0 0 0 E 0 0 0 90000001		; VP overflow
@	LOC	1       1     E	0			; Missing N or S
@	LOC	1     x 1     E	0			; Bad letter
@	LOC	1     N 1      	0			; Missing E or W
@	LOC	1     N 1     x	0			; Bad letter
@	LOC	1     N 1     E				; Missing altitude
@	LOC	0 0 0 N 0 0 0 E 0 0 0 0 x		; Unexpected item
@	LOC	\# 17 00 00 00 00 00000000 00000000 00000000 00	; Too long rdata
@	LOC	\# 17 00 00 00 00 00000000 00000000 00000000	; Bad rdata length
