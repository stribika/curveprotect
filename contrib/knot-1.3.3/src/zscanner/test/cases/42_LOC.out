OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=001216138036EE808036EE8000989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=001216138000EA608036EE8000989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=00121613800003E88036EE8000989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=001216138036EE808000EA6000989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=001216138036EE80800003E800989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=001216138036EE80800003E800989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=001216138036EE808036EE8000989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=001216138036EE808036EE8000989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=000012138036EE808036EE8000989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=000012138036EE808036EE8000989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=000000128036EE808036EE8000989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=000000128036EE808036EE8000989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=00000000800000008000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=009999996C79388159295F81FFFFFFFF
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=00121613800000007FFFFFFF00989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=00121613800000007FFFFFF600989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=00121613800000007FFFFF9C00989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=00121613800000007FFFFC1800989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=00121613800000007FFFD8F000989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=00101613800000008000000000989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=00111613800000008000000000989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=00121613800000008000000000989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=00131613800000008000000000989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=00141613800000008000000000989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=00151613800000008000000000989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=00161613800000008000000000989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=00171613800000008000000000989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=00181613800000008000000000989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=00191613800000008000000000989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=00000000000000000000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=00000000000000000000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=00121613800000008000000000989680
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=001D
RDATA=00121613800000008000000000989680
------
WARNG=ZSCANNER_EBAD_LOC_DATA
------
WARNG=ZSCANNER_EBAD_LOC_DATA
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_LOC_DATA
------
WARNG=ZSCANNER_EBAD_LOC_DATA
------
WARNG=ZSCANNER_EBAD_LOC_DATA
------
WARNG=ZSCANNER_EBAD_LOC_DATA
------
WARNG=ZSCANNER_EBAD_LOC_DATA
------
WARNG=ZSCANNER_EBAD_LOC_DATA
------
WARNG=ZSCANNER_EBAD_HEX_RDATA
------
WARNG=ZSCANNER_EBAD_RDATA_LENGTH
------
